src/Plugin/WebformElement/NetsEasyPaymentElement.php

```diff
@@ -126,4 +134,29 @@ public function alterForm(array &$element, array &$form, FormStateInterface $for
     }
   }
 
+  /**
+   * Get recipient elements.
+   *
+   * @phpstan-return array<string, mixed>
+   */
+  private function getRecipientElements(): array {
+    $elements = $this->getWebform()->getElementsDecodedAndFlattened();
+
+    $elementTypes = [
+      'textfield',
+      'hidden',
+      'select',
+    ];
+    $elements = array_filter(
+      $elements,
+      static function (array $element) use ($elementTypes) {
+        return in_array($element['#type'], $elementTypes, TRUE);
+      }
+    );
+
+    return array_map(static function (array $element) {
+      return $element['#title'];
+    }, $elements);
```

Comment:
We love [arrow functions](https://www.php.net/manual/en/functions.arrow.php):

```suggestion
    $elements = array_filter(
      $elements,
      static fn(array $element) => in_array($element['#type'], $elementTypes, TRUE)
    );

    return array_map(
      static fn(array $element) => $element['#title'],
      $elements
    );
```

The final map should probably just use [`array_column`](https://www.php.net/manual/en/function.array-column.php) – another function we love:

```php
    return array_column($elements, ['#title']);
```